src/core/lib/gprpp/time.h

```diff
@@ -38,7 +38,8 @@
     uint64_t now = grpc_core::Timestamp::FromTimespecRoundDown( \
                        gpr_now(GPR_CLOCK_MONOTONIC))            \
                        .milliseconds_after_process_epoch();     \
-    if ((now - prev.exchange(now)) > (n)*1000) {                \
+    if ((now - prev) > (n)*1000) {                              \
```

Comment:
I suggest reverting this change, explicit atomic operations are preferred in this codebase, and we use `atomic::exchange` a fair amount. This change removes an explicit call, and adds an implicit load and store.

Reply:
It looks like https://github.com/grpc/grpc/pull/32204 fixes the bug as well, and only uses one `exchange`. That will likely land today. Thanks for catching the bug!